File: 240704/ab 사이에 없는 c/c-between-a-and-b-2.py
arr = input().split()
a,b,c = int(arr[0]), int(arr[1]), int(arr[2])

satisfied = True #응 다 없어.. 
for i in range (a,b+1):
    if i % c == 0 :
        satisfied = False # 하나는 있다 얘

if satisfied == False :
    print("NO") #그래 있다고
else:
    print("YES") # 없당께

File: 241125/ab 사이에 없는 c/c-between-a-and-b-2.c
#include <stdio.h>
#include <stdbool.h>

//다 만족한다는 전제 하에 만약에 하나라도 틀린게 나오면 그때 NO로 진행

int main() {
    // 여기에 코드를 작성해주세요.
    int a,b,c;

    scanf("%d %d %d", &a, &b, &c);

    bool satisfied = true; //a~b 사이에 c의 배수가 없음. 만약 있으면 NO

    for (int i=a; i<=b; i++){
        if (i%c==0){
            satisfied = false; 
        }
    }

    if (satisfied == false) {
        printf("NO");
    }
    else {
        printf("YES");
    }
    return 0;
}